docs/community/tutorials-contest/2022/avax-subnet-customization/README.md

```diff
@@ -0,0 +1,975 @@
+
+![Compile](.github/images/compiling.png)
+
+Go to the deploy tab, choose injected web3 as enviroment - connect your wallet - then choose Game in the contract dropdown. Then click the deploy button.
+
+![Deploy](.github/images/deploying.png)
+
+Confirm the transaction with your metamask wallet.
+
+![Deployed contracts](.github/images/game_deployed.png)
+
+Yay! We deployed a smart contract in our Subnet! That sounds fantastic!
+
+## Setting Game Contract as a Minter
+
+We're admin in [`contractNativeMinterConfig`](#contractnativeminterconfig).
+Which means we can add new Minters to the network using `NativeMinterInterface`.
+
+We already created `NativeMinterInterface.sol`
+
+Go to remix and compile `NativeMinterInterface.sol` by following the same steps as `Game.sol`.
+
+Go to the deploy tab in remix, and choose NativeMinterInterface in the contract dropdown. Then type in the address `0x0200000000000000000000000000000000000001` to `Load contract from Address` then click `At Address`
+
+![Interface steps](.github/images/interface.png)
+
+After clicking the button, you should see the `NATIVEMINTERINTERFACE AT 0X02..` under the `Deployed Contracts` section.
+
+We will call the `setEnabled` function of `NativeMinter` contract with the address of `Game` contract to set the `Game` contract as a minter.
+
+![Set minter](.github/images/setenabled.png)
+
+Confirm the transaction with your metamask wallet. We can call this function because we've added our address as an admin in the `contractNativeMinterConfig` section.
+
+Now the `Game.sol` contract is a minter and can mint native tokens!
+
+## Play the Game
+
+In the deploy tab: 
+- Set the value to 100 Ether or any other amount you want to play with.
+- Find `Game` contract under deployed contracts section in remix.
+- Call `play` function.
+- Confirm the transaction with your metamask wallet.
+
+![play](.github/images/play.png)
+
+Then check your metamask wallet.
+
+![result](.github/images/lucky.png)
+
+I first played with 10 and then 100 tokens. Seems like it's my lucky day! I already earned 110 LUCK tokens. As you can see, our smart contract **can mint native tokens!**
+
+**Let's go all in!**
+
+![allin](.github/images/outofluck.png)
+
+I'm out of **luck**!
+
+As an admin in `NativeMinter` contract, I can mint native tokens as well! Let's mint ourselves 1000 LUCK tokens.
+
+![minting native tokens](.github/images/mintingtomyself.png)
+
+And my luck is back!
+
+![luck back](.github/images/luckisback.png)
+
+# Conclusion
+This is the end of the tutorial. I hope you've enjoyed it as much as I've enjoyed writing it. Subnets are literally amazing and I'm very excited for the future of this technology.
+
+You now have the ability to create your own Subnet and customize it to your needs. Go play with the parameters, try something new, and enjoy!
+
+Please feel free to create an issue if you have any questions!
+
+Thanks for reading :heart:
+
+## Resources
```

Comment:
Per [Markdownlint](https://github.com/DavidAnson/markdownlint) Headings should be surrounded by blank lines.

> MD022/blanks-around-headings/blanks-around-headers: Headings should be surrounded by blank lines [Expected: 1; Actual: 0; Below]

[More info: MD022](https://github.com/DavidAnson/markdownlint/blob/v0.25.1/doc/Rules.md#md022---headings-should-be-surrounded-by-blank-lines)